index.html

```diff
@@ -9,13 +9,39 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
-    <header>
+    <header class="container">
+      <h1 id="logo">Aziz Mejri</h1>
+      <nav class="navbar">
+        <button type="button" class="hamburger-menu">
+          <span class="line line-1"></span>
+          <span class="line line-2"></span>
+          <span class="line line-3"></span>
+        </button>
+
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a class="nav-link" href="#projects">Portoflio</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#about-me">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- <header>
       <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
-      </button>
       <nav id="desktop-nav">
-        <ul>
+         <button type="button" class="menu-bars">
+           <i class="fas fa-bars"></i>
+         </button>
+      
+        <ul class="nav-item">
```

Comment:
- [ ] It would be better if you remove unnecessary comments to have a readable and clean code. 

Please check the [HTML&CSS best practices in the curriculum.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.)